frontend/fingertips-frontend/components/organisms/BarChartEmbeddedTable/index.tsx

```diff
@@ -97,55 +72,55 @@ export function BarChartEmbeddedTable({
               {BarChartEmbeddedTableHeadingEnum.AreaName}
             </Table.CellHeader>
             <Table.CellHeader>
-              {BarChartEmbeddedTableHeadingEnum.AreaCount}
+              {BarChartEmbeddedTableHeadingEnum.Count}
             </Table.CellHeader>
             <Table.CellHeader>
-              {BarChartEmbeddedTableHeadingEnum.AreaValue} %
+              {BarChartEmbeddedTableHeadingEnum.Value} {measurementUnit}
             </Table.CellHeader>
             <Table.CellHeader>
-              {BarChartEmbeddedTableHeadingEnum.AreaLower}
+              {BarChartEmbeddedTableHeadingEnum.Lower}
             </Table.CellHeader>
             <Table.CellHeader>
-              {BarChartEmbeddedTableHeadingEnum.AreaUpper}
+              {BarChartEmbeddedTableHeadingEnum.Upper}
             </Table.CellHeader>
           </Table.Row>
         }
       >
         {mostRecentBenchmarkData ? (
           <Table.Row
-            key={`${mostRecentBenchmarkData.area}`}
+            key={`${benchmarkData?.areaName}`}
             style={{ backgroundColor: GovukColours.MidGrey }}
             data-testid="table-row-benchmark"
           >
-            {checkIfValueExists(mostRecentBenchmarkData.area)}
-            {checkIfValueExists(mostRecentBenchmarkData.count)}
-            {checkIfValueExists(mostRecentBenchmarkData.value)}
-            {checkIfValueExists(mostRecentBenchmarkData.lowerCi)}
-            {checkIfValueExists(mostRecentBenchmarkData.upperCi)}
+            <CheckValueInTableCell value={benchmarkData?.areaName} />
+            <CheckValueInTableCell value={mostRecentBenchmarkData.count} />
+            <CheckValueInTableCell value={mostRecentBenchmarkData.value} />
+            <CheckValueInTableCell value={mostRecentBenchmarkData.lowerCi} />
+            <CheckValueInTableCell value={mostRecentBenchmarkData.upperCi} />
           </Table.Row>
         ) : null}
 
         {mostRecentGroupData ? (
           <Table.Row
-            key={`${mostRecentGroupData.area}`}
+            key={`${mostRecentGroupData.count}`}
```

Comment:
I don't think there's any guarantee the `count` value is going to be unique across areas. Given this is always a single row, can't you just give it a fixed key?
```suggestion
            key="group-data"
```

Reply:
It was supposed to be groupIndicatorData?.areaName, Ill change it back

Reply:
OK, that works too.